doc/release-notes/release-notes-bucash1.3.0.0.md

```diff
@@ -0,0 +1,230 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.3.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.3.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a major release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specifications you could find here:
+
+https://github.com/bitcoincashorg/spec/blob/master/uahf-technical-spec.md (Aug 1st Protocol Upgrade, bucash 1.1.0.0)
+https://github.com/bitcoincashorg/spec/blob/master/nov-13-hardfork-spec.md (Nov 13th Protocol Upgrade, bucash 1.1.2.0)
+https://github.com/bitcoincashorg/spec/blob/master/may-2018-hardfork.md (May 15th Protocol Upgrade, bucash 1.3.0.0)
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+If you are upgrading from a release older than 1.1.2.0, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Downgrade
+---------
+
+In case you decide to downgrade from BUcash 1.2.0.1 to a version older than 1.1.2.0
+will need to run the old release using `-reindex` option so that the
```

Comment:
nit:  this should probably be  "from BUcash 1.2.0.1, or greater, to a version..."

Reply:
agreed

Reply:
change pushed